src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -347,34 +327,34 @@ function WorkspaceMembersPage(props) {
             // If this policy is owned by Expensify then show all support (expensify.com or team.expensify.com) emails
             // We don't want to show guides as policy members unless the user is a guide. Some customers get confused when they
             // see random people added to their policy, but guides having access to the policies help set them up.
-            if (PolicyUtils.isExpensifyTeam(details.login || details.displayName)) {
+            if (PolicyUtils.isExpensifyTeam(details?.login ?? details?.displayName ?? '')) {
```

Comment:
Is it possible for login to be an empty string?

Reply:
Probably not, I made the argument optional as the implementation already treated it as it could be undefined.